### ISSUE TITLE
Stepper: next button label includes header text

### ISSUE THREAD

**Issue by ajay1208** [angular/components#16166]
![Accesasibility-stepper issue](https://user-images.githubusercontent.com/50059018/58664309-fd4f3080-82f3-11e9-8dcb-a42b11c9a709.png)
#### Reproduction

https://stackblitz.com/angular/rdypvkkpelo?file=app%2Fstepper-overview-example.ts

Steps to reproduce:
1. Add new header with any heading tag inside the stepper step.
2. Then run the jaws and if we check by using insert + F3 and then button list. All the buttons in the page are appending with the header.
 
 
#### Expected Behavior

What behavior were you expecting to see?

Button with its own name

#### Actual Behavior

What behavior did you actually see?
 header then button name

**Comment by ajay1208** [comment]
Any update on this issue?

**Comment by rajudevansh** [comment]
am also facing mentioned issue in mat stepper. Any information will be helpful for us.

**Comment by rchjava** [comment]
i am also facing issue

**Comment by ajay1208** [comment]
josephperrott Any update on this?

**Comment by ajay1208** [comment]
@josephperrott Can this be fixed?

**Comment by ajay1208** [comment]
@crisbeto Can you please take a look at this issue.

**Comment by jelbourn** [comment]
I looked at this just now, and the button label seems normal:
![image](https://user-images.githubusercontent.com/838736/100173049-8a8a2b80-2e7e-11eb-9076-846324568f19.png)

Can you share a reproduction of what you're seeing? The linked stackblitz does not have any headers.

**Comment by jelbourn** [comment]
Closing since this seems to be working as expected.
